variables.tf

```diff
@@ -276,6 +276,17 @@ variable "deployment_minimum_healthy_percent" {
   default     = 100
 }
 
+variable "availability_zone_rebalancing" {
+  type        = string
+  description = "ECS automatically redistributes tasks within a service across Availability Zones (AZs) to mitigate the risk of impaired application availability due to underlying infrastructure failures and task lifecycle activities. The valid values are `ENABLED` and `DISABLED`."
+  default     = "DISABLED"
```

Comment:
Would it be better to set the default to null instead?

Reply:
@nitrocode done

Reply:
```suggestion
  default     = null
```